src/app/core/common-components/entity-select/entity-select.component.ts

```diff
@@ -69,7 +69,7 @@ export class EntitySelectComponent<
     }
 
     this._entityType = Array.isArray(type) ? type : [type];
-    this.loadAvailableEntities();
+    this.loadAvailableEntities().then((_) => {});
```

Comment:
Is this required? (and what effect does it have? Promises should always execute, not only upon subscription, I think?)